components/nominations/NomineeCard.tsx

```diff
@@ -1,9 +1,15 @@
+import { Clickable } from "components/types";
+import { Nominee } from "data/types";
```

Comment:
```suggestion
import { Clickable } from "../../components/types";
import { Nominee } from "../../data/types";
```
Vscode wouldn't let me hover over Clickable & get type info without it (plus, had red underlines)

Reply:
Again, weird - my VScode works find + yarn build works 🤔